### PR TITLE
Func note is copied same as other one

### PR DESCRIPTION
**Pull request by lixiaobing1**
Func note is copied same as other one

### CODE REVIEW

File: pkg/controller/service/servicecontroller.go [hunk 1]
```diff
@@ -443,8 +443,7 @@ func (s *serviceCache) GetByKey(key string) (interface{}, bool, error) {
 	return nil, false, nil
 }
 
-// ListKeys implements the interface required by DeltaFIFO to list the keys we
-// already know about.
+// allServices returns all services cached.
```
Comment (mikedanese) [inline pkg/controller/service/servicecontroller.go]: can we just remove the comment altogether?

**Comment by lixiaobing10051267** [comment]
@mikedanese ok, the func note has been deleted.

**Comment by MHBauer** [comment]
change looks good.

**Comment by mikedanese** [comment]
LGTM, please squash commits.

**Comment by lixiaobing10051267** [comment]
@mikedanese commits have been squashed:)

**Comment by mikedanese** [comment]
LGTM

**Pull request merged**